packages/analytics-js/src/components/configManager/ConfigManager.ts

```diff
@@ -115,7 +117,12 @@ class ConfigManager implements IConfigManager {
         state.consents.activeConsentManagerPluginName.value = consentManagerPluginName;
 
         // set storage type in state
-        state.storage.type.value = state.loadOptions.value.storage?.type;
+        const storageType = state.loadOptions.value.storage?.type;
+        if (!isValidStorageType(storageType)) {
+          this.onError(new Error(STORAGE_TYPE_VALIDATION_ERROR(storageType)));
+          return;
```

Comment:
Shouldn't we default to `cookie` type and turn this into a warning?

Reply:
> cookie type and turn this into a warning
Already done in store manager module.

Reply:
Done